### ISSUE TITLE
Saving layout throws error

### ISSUE THREAD

**Issue by brapp96** [spyder-ide/spyder#15192]
### Issue Report Checklist

* [x] Searched the [issues page](https://github.com/spyder-ide/spyder/issues?q=is%3Aissue) for similar reports
* [x] Read the relevant sections of the [Spyder Troubleshooting Guide](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ) and followed its advice
* [x] Reproduced the issue after updating with ``conda update spyder`` (or ``pip``, if not using Anaconda)
* [x] Could not reproduce inside ``jupyter qtconsole`` (if console-related)
* [x] Tried basic troubleshooting (if a bug/error)
    * [x] Restarted Spyder
    * [x] Reset preferences with ``spyder --reset``
    * [x] Reinstalled the latest version of [Anaconda](https://www.anaconda.com/download/)
    * [x] Tried the other applicable steps from the Troubleshooting Guide
* [x] Completed the **Problem Description**, **Steps to Reproduce** and **Version** sections below

## Problem Description
Saving a layout throws an error. I tried to change the layout at first, but saving the default layout with a new name also triggered the same error. I just installed 5.0.0, but I don't remember ever getting this to work with 4 either, though it didn't throw an error.

Noting additionally that I wasn't able to report the error through the error tracker, which threw a 404 error. Perhaps there was a naming change there due to the version switch?

### What steps reproduce the problem?

1. Open spyder
2. View->Window layouts->Save current layout
3. Give it any name and press Okay

### What is the expected output? What do you see instead?

Expected: layout is saved and I can switch to it
Instead: spyder throws an error

### Paste Traceback/Error Below (if applicable)

```
Traceback (most recent call last):
  File "/home/brian/.local/lib/python3.8/site-packages/spyder/plugins/layout/container.py", line 131, in <lambda>
    triggered=lambda: self.show_save_layout(),
  File "/home/brian/.local/lib/python3.8/site-packages/spyder/plugins/layout/container.py", line 307, in show_save_layout
    self.save_current_window_settings('layout_{}/'.format(index),
AttributeError: 'LayoutContainer' object has no attribute 'save_current_window_settings'
```

## Versions

* Spyder version: 5.0.0 None
* Python version: 3.8.5 64-bit
* Qt version: 5.12.9
* PyQt5 version: 5.12.3
* Operating System: Linux 5.4.0-70-generic

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0          :  1.4.0 (OK)
chardet >=2.0.0               :  3.0.4 (OK)
cloudpickle >=0.5.0           :  1.6.0 (OK)
cookiecutter >=1.6.0          :  1.7.2 (OK)
diff_match_patch >=20181111   :  20200713 (OK)
intervaltree >=3.0.2          :  3.1.0 (OK)
IPython >=7.6.0               :  7.22.0 (OK)
jedi =0.17.2                  :  0.17.2 (OK)
jsonschema >=3.2.0            :  3.2.0 (OK)
keyring >=17.0.0              :  18.0.1 (OK)
nbconvert >=4.0               :  6.0.7 (OK)
numpydoc >=0.6.0              :  1.1.0 (OK)
parso =0.7.0                  :  0.7.0 (OK)
pexpect >=4.4.0               :  4.8.0 (OK)
pickleshare >=0.4             :  0.7.5 (OK)
psutil >=5.3                  :  5.7.2 (OK)
pygments >=2.0                :  2.8.1 (OK)
pylint >=1.0                  :  2.6.0 (OK)
pyls >=0.36.2;<1.0.0          :  0.36.2 (OK)
pyls_black >=0.4.6            :  0.4.6 (OK)
pyls_spyder >=0.3.2           :  0.3.2 (OK)
qdarkstyle =3.0.2             :  3.0.2 (OK)
qstylizer >=0.1.10            :  0.1.10 (OK)
qtawesome >=0.5.7             :  1.0.1 (OK)
qtconsole >=5.0.3             :  5.0.3 (OK)
qtpy >=1.5.0                  :  1.9.0 (OK)
setuptools >=39.0.0           :  54.2.0 (OK)
sphinx >=0.6.6                :  3.2.1 (OK)
spyder_kernels >=2.0.1,<2.1.0 :  2.0.1 (NOK)
textdistance >=4.2.0          :  4.2.0 (OK)
three_merge >=0.1.1           :  0.1.1 (OK)
watchdog >=0.10.3;<2.0.0      :  0.10.3 (OK)
xdg >=0.26                    :  0.26 (OK)
zmq >=17                      :  22.0.3 (OK)

# Optional:
cython >=0.21                 :  None (NOK)
matplotlib >=2.0.0            :  3.3.2 (OK)
numpy >=1.7                   :  1.18.5 (OK)
pandas >=1.1.1                :  1.2.3 (OK)
scipy >=0.17.0                :  1.5.2 (OK)
sympy >=0.7.3                 :  1.7.1 (OK)
```

**Comment by ccordoba12** [comment]
Hey @brapp96, thanks for reporting. This is a duplicate of issue #15129 and it'll be fixed in our next version (5.0.1), to be released shortly.

**Comment by brapp96** [comment]
Oh, sorry, guess I missed that one. Thanks for the response!
